spec.html

```diff
@@ -74,10 +74,7 @@ <h1>Runtime Semantics: PerformPromiseAny ( _iteratorRecord_, _constructor_, _res
           1. Set _iteratorRecord_.[[Done]] to *true*.
           1. Set _remainingElementsCount_.[[Value]] to _remainingElementsCount_.[[Value]] - 1.
           1. If _remainingElementsCount_.[[Value]] is 0, then
-            1. Let _errorsArray_ be CreateArrayFromList(_errors_).
-            1. Let _error_ be a newly created `AggregateError` object.
-            1. Perform ! CreateDataProperty ( _error_, `"errors"`, _errorsArray_ ).
-            1. Return ThrowCompletion ( _error_ ).
+            1. Return ThrowAggregateError(_errors_).
```

Comment:
An alternative would be `Perform ? ThrowAggregateError`, which might better match the intended usage and also the naming (ie, `throw AggregateError`)

Reply:
Does `Perform` also return a value?

Reply:
No, but the `?` will ReturnIfAbrupt.

Reply:
Hm, I prefer `Return ThrowAggregateError(_errors_)`; it seems a little silly to have a `If argument is an abrupt completion` test when you know that it will always be an abrupt completion. But whatever; I'll defer to the editors.